### PR TITLE
[DPR-API #46] Data Proxy for CSV and JSON

### PR DESCRIPTION
**Pull request by subhankarb**

### CODE REVIEW

File: app/mod_api/controllers.py [hunk 1]
```diff
@@ -171,6 +171,57 @@ def get_metadata(publisher, package):
         return handle_error('GENERIC_ERROR', e.message, 500)
 
 
+@mod_api_blueprint.route("/dataproxy/<publisher>/<package>/<resource>.json", methods=["GET"])
```
Comment (pwalsh) [inline app/mod_api/controllers.py]: @subhankarb @rgrp 

why handle json and csv explicitly? What if someone adds a non-tabular data package, or, if at some point we support data package descriptors in yaml etc.? I get that we want to spike a solution and therefore we might base our implementation around *tabular* data package, but the hard coding of format here seems unnecessarily limiting.

Comment (zelima) [inline app/mod_api/controllers.py]: @pwalsh For now this is done for frontend javascript to handle with graphs and tables etc... 
We can continue discussions in #46 
Merging for now.

Comment (pwalsh) [inline app/mod_api/controllers.py]: @zelima my comment is not related to what a client does with data from this endpoint. It is related to the reduced capabilities of the endpoint via hardcoding of file type information, unecessarily I think. It is not the type of thing that should stop merging, but definitely the type of thing to note as an issue in the backlog or something, for the implementing team to keep visibility on it.

Comment (rgrp) [inline app/mod_api/controllers.py]: @pwalsh i totally agree but note this is a spike solution porting the current solution from data.okfn.org.

**Pull request merged**